docs/identities/get-started/account-recovery.mdx

```diff
@@ -42,11 +42,23 @@ configure account recovery in your Ory project.
 </BrowserWindow>
 ```
 
-**Recovery strategy**
+### Enable the feature flag `choose_recovery_address` (unlocks sending a recovery code via SMS)
+
+1. Log in to your [Ory Console](https://console.ory.sh/)
+2. Select your workspace and project
+3. Navigate to **Project settings** → **Advanced**
```

Comment:
If you use `<ConsoleLink />` component you don't need to manually add the Page section and title here, and it will be auto updated if we ever change titles in the future. (See [here](https://github.com/ory/docs/blob/ceceb67a24c7c974327fc8c67844440e3765659e/docs/identities/sign-in/identifier-first-authentication.mdx?plain=1#L22-L28) for an example.)

And one last thing, how would one use the CLI to enable this? IMO, this should use the `<Tabs>` component (search the ory/docs repo for usage examples), which has both "Ory Network" and "Ory CLI" as options, and describes the process for either.